.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+
+jdk:
+  - oraclejdk8
+
+services:
+  - docker
```

Comment:
this can be removed after docker images are not built anymore, we'll have builds running on containerized environments (docker) which spin up faster